com.unity.netcode.gameobjects/Runtime/Messaging/Messages/ProxyMessage.cs

```diff
@@ -63,7 +63,7 @@ public unsafe void Handle(ref NetworkContext context)
             using (WrappedMessage.WriteBuffer)
             {
                 WrappedMessage.WriteBuffer.WriteBytesSafe(WrappedMessage.ReadBuffer.GetUnsafePtr(), WrappedMessage.ReadBuffer.Length);
-                networkManager.MessageManager.SendMessage(ref WrappedMessage, Delivery, nonServerIds);
+                networkManager.MessageManager.SendMessage(ref WrappedMessage, Delivery, nonServerIds.AsArray());
```

Comment:
I actually think `NetworkMessageManager.SendMessage` should be adjusted to support a `NativeList` overload rather than changing this to pass `AsArray()`. It could basically be a direct copy-paste of the NativeArray overload (which is only one line anyway).